modules/ossm-deploying-automatic-gateway-injection.adoc

```diff
@@ -73,9 +82,68 @@ roleRef:
 subjects:
 - kind: ServiceAccount
   name: default
+    ---
+    apiVersion: networking.k8s.io/v1
+    kind: NetworkPolicy <6>
+    metadata:
+      name: vs2igw
+      namespace: vs2
+    spec:
+      podSelector:
+        matchLabels:
+          istio: ingressgateway
+      ingress:
+        - {}
+      policyTypes:
+      - Ingress
+    ---
```

Comment:
Please remove 4 leading spaces in lines 85-99.

Reply:
done